reference/release-notes/1.23.0.md

```diff
@@ -0,0 +1,92 @@
+---
+orphan: true
+---
+# 1.23.0
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.23.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Anbox Cloud Appliance
+The Anbox Cloud Appliance is overhauled to deliver a much simpler and streamlined version. This new version of the appliance is made available using the [snap epoch feature](https://snapcraft.io/docs/snap-epochs). Existing installations of the appliance are not affected but the old snap with `epoch=0` is {ref}`deprecated <sec-appliance-deprecation>`. For all new installations of the appliance, use the `epoch=1` snap. See the [appliance installation tutorial](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/) for instructions.
+
+The `anbox-cloud-appliance` status command now displays the initialisation status of the appliance and its version. <!--AC-2513-->
+
+### Seamless ADB connections
+
+Anbox Cloud now offers a way to connect to remote Android instances securely using the Android Debug Bridge (ADB). You can set up an ADB connection using the dashboard or the CLI. See {ref}`howto-access-instance` for more information.
+
+### Full support for AAOS
+
+Android Automotive OS based images are fully supported starting from this release and no longer considered an experimental feature.<!--AC-2660-->
+
+### Anbox Cloud dashboard
+
+The Anbox Cloud dashboard has the following improvements:
+
+  * The *Instances* and the *Sessions* pages and their functionalities are merged to provide an enhanced user experience while streaming.
+
+  * The initial VHAL values upon joining a stream are now captured as a default. You can revert to these values using the *Reset* button, when needed.
+
+  * You can set up and authorise ADB connections for an instance from the dashboard.
+
+### Anbox Cloud charms
+
+All Anbox Cloud charms now install Snaps directly from the Snap Store instead of using deb packages. Existing installations automatically switch to snap refresh from the store on upgrade.<!--AC-2523-->
+
+### Vendor image
+
+The Anbox vendor images are now based on Android 14. Building on the latest version of Android helps us provide the latest updates and security fixes.
+
+### Security updates
+
+Android security updates for August 2024 (see [Android Security Bulletin - August 2024](https://source.android.com/docs/security/bulletin/2024-08-01) for more information).
+
+### WebView updates
+
+The Android WebView has been updated to [127.0.6533.84](https://chromereleases.googleblog.com/2024/07/chrome-for-android-update_30.html).
```

Comment:
The version of Android webview shipped into 1.23 lxd image is `126.0.6478.72`

Reply:
Have asked @morphis to confirm. According to https://chromiumdash.appspot.com/releases?platform=Android, 127.0.6533.84 is the latest completed build.

Reply:
+1, thanks for the correction